pkg/utils/database/postgres.go

```diff
@@ -236,8 +299,14 @@ func (p Postgres) deleteDatabase(admin AdminCredentials) error {
 
 func (p Postgres) deleteUser(admin AdminCredentials) error {
 	delete := fmt.Sprintf("DROP USER \"%s\";", p.User)
+	dropOwned := fmt.Sprintf("DROP OWNED BY \"%s\";", p.User)
 
 	if p.isUserExist(admin) {
+		logrus.Info("DROPPING OWNED")
+		if err := p.executeQuery("postgres", dropOwned, admin); err != nil {
+			logrus.Errorf("Can't drop owned by user \"%s\" %s", p.User, err)
+			return err
+		}
 		err := p.executeQuery("postgres", delete, admin)
```

Comment:
same here

Reply:
I don't think I have added this code and so I'm not sure that I should update it within this ticket